test/powershell/Language/Scripting/I18n.Tests.ps1

```diff
@@ -52,6 +52,7 @@ Describe 'Testing of script internationalization' -Tags "CI" {
 
         import-localizedData mydata -uiculture nl-NL -ea SilentlyContinue -ev ev
 
+        $ev | Should Not BeNullOrEmpty
```

Comment:
$ev will always be an arraylist, but it might be empty. I think here you want to test `$ev[0] | Should Not BeNullOrEmpty`.